src/tests/ftest/util/command_utils.py

```diff
@@ -1088,15 +1083,17 @@ def verify_socket_directory(self, user, hosts):
                 self.log.info(
                     "%s: creating socket directory %s for user %s on %s",
                     self.command, directory, user, nodes)
-                try:
-                    create_directory(nodes, directory, sudo=True)
-                    change_file_owner(nodes, directory, user, get_primary_group(user), sudo=True)
-                except DaosTestError as error:
+                result = create_directory(self.log, nodes, directory, user="root")
+                if not result.passed:
+                    raise CommandFailure(
+                        f"{self.command}: error creating socket directory {directory} for user "
+                        f"{user} on {result.failed_hosts}")
```

Comment:
IMO it's overkill to print the command and also the directory, user, hosts. Especially at large scale, it's redundant and cluttered

Reply:
If there is a failure there would only be one of these messages and I think it would be useful to know at least on which hosts it failed.

Reply:
But we already know the hosts it failed on because it's printed immediately above.

Reply:
In the log yes, but this gives us a concise message to include in the results.xml error summary.

Reply:
It's not concise at large scale. It's huge :)